libs/components/charts/src/lib/chart.html

```
@@ -0,0 +1,15 @@
+<div class="sky-chart-header sky-margin-stacked-sm">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)